server/routes/api/v1/petTypesRouter.js

```diff
@@ -1,17 +1,31 @@
 import express from "express"
 
-import PetType from "../../../Models/PetType.js"
+import PetType from "../../../models/PetType.js"
+import AdoptionApplication from "../../../models/AdoptionApplication.js"
+
 
 const petTypesRouter = new express.Router();
 
 petTypesRouter.get("/", async (req, res) => {
-    try{
+    try {
         const petTypes = await PetType.findAll();
-        res.status(200).json({petTypes: petTypes})
-    } catch(err) {
+        res.status(200).json({ petTypes: petTypes })
+    } catch (err) {
+        console.log(err);
+        res.status(500).json({ error: err });
+    }
+})
+
+petTypesRouter.post("/", async (req, res) => {
+    try {
+        const applicationObject = new AdoptionApplication(req.body)
```

Comment:
Is the Adoption Application for submitting a new Pet Type? If not, we probably want the application to be POSTed and created in a router specific for Adoption Applications